system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -235,7 +303,9 @@ class TopSitesEdit extends React.Component {
                   key={link.guid || link.url}
                   dispatch={this.props.dispatch}
                   link={link}
-                  index={index} />)}
+                  index={index}
+                  intl={this.props.intl}
+                  editMode={true} />)}
```

Comment:
This is out of scope for this ticket, but maybe we can file a follow up ticket to factor our the modal and some of the edit logic into it's own component so the top sites component doesn't get too gnarly. It would be nice if we only had one set of top sites being rendered at a time, and they could either be in edit mode or not, rather than rendering the default top sites and then rendering the editable top sites on top. Can you file a ticket for that?